canvas/template.html

```diff
@@ -0,0 +1,34 @@
+#   Copyright 2017 Google Inc. All Rights Reserved.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+function GetVariable(fuzzervars, var_type) { if(fuzzervars[var_type]) { return fuzzervars[var_type]; } else { return null; }}
+
+function SetVariable(fuzzervars, var_name, var_type) { fuzzervars[var_type] = var_name; }
+
+function draw() {
+
+    var canvas = document.createElement('canvas');
+
+    canvas.id = "CursorLayer";
+    canvas.width = 200;
+    canvas.height = 200;
+    canvas.style.position = "absolute";
+    canvas.style.border = "1px solid";
+
+    var body = document.getElementsByTagName("body")[0];
+    body.appendChild(canvas);
+
+    var ctx = canvas.getContext('2d');
+
+    try { <canvasfuzz>; } catch(e) { console.log(e.message) }

```

Comment:
why have try/catch around `<canvasfuzz>` when each line of `canvasfuzz` is also going to be surrounded by try/catch?

Reply:
If you notice from my grammar:
#!lineguard try { <line>; } catch(e) { console.log(e.message) }
this line is actually commented out, hence it will never run.
I removed it :)